aries-site/src/data/featured.js

```diff
@@ -0,0 +1,23 @@
+import { Brush, Code, Star } from 'grommet-icons';
+
+export const featured = [
+  {
```

Comment:
Any way this could be simplified to an array of strings: ['Designer Guidance', 'Developer Guidance', 'What's New']?

To get the data, we reference these names from the existing data structures in extend.js, etc. and is pulled from there? Aiming to have on source of truth for page names, descriptions, icons.

Icon would need to be added to the data structure there, but I think that is fine.

Reply:
Seems reasonable in this case.